page/using-jquery-core/faq/how-do-i-select-an-element-by-an-id-that-has-characters-used-in-css-notation.md

```diff
@@ -20,12 +20,14 @@ $( "#some.id" )
 $( "#some\\.id" )
 ```
 
-The following function takes care of escaping these characters and places a "#" at the beginning of the ID string:
+In case the jQuery library in use is sufficiently recent, the function [jQuery.escapeSelector()](https://api.jquery.com/jQuery.escapeSelector/) should be used to treat such selectors.
```

Comment:
Since we introduced it in 3.0, how about if we save people a click and mention that here?

Reply:
Did it a year ago and now it turns out I should have "resolved" this conversation? Well, now I do :grin: Updated the pull-request (cleaned up commits, rebased onto current master).